lib/elixir/lib/supervisor/spec.ex

```diff
@@ -224,6 +224,17 @@ defmodule Supervisor.Spec do
 
   Check `Supervisor.Spec` module docs for more information on
   the options.
+
+  ## Examples
+
+  Here we start a `GenEvent` that is named `MyGenEvent`. We test that the
+  GenEvent is actually created with that name by calling a function that
+  `which_handlers`. As no handlers are added, it returns an empty list.
+
+      iex> genevent_worker = Supervisor.Spec.worker(GenEvent, [[name: MyGenEvent]])
```

Comment:
@fishcakez there is a note about `:dynamic` on line#71:
https://github.com/Havvy/elixir/blob/spec-docs/lib/elixir/lib/supervisor/spec.ex#L71.